docs/downloads/automation-library/integrations/pydoc/review_python_large_changes.cm

```diff
@@ -0,0 +1,33 @@
+# -*- mode: yaml -*-
+manifest:
+  version: 1.0
+
+automations:
+  #Require more extensive reviews for large Python changes that lack Pydoc updates.
+  review_python_large:
+    if:
+      - {{ changes.ratio > 25}}
```

Comment:
Change this to if the number of changed lines is larger than 150, not the ratio.